pkg/postage/stamper.go

```diff
@@ -39,13 +40,13 @@ func NewStamper(store storage.Store, issuer *StampIssuer, signer crypto.Signer)
 
 // Stamp takes chunk, see if the chunk can be included in the batch and
 // signs it with the owner of the batch of this Stamp issuer.
-func (st *stamper) Stamp(addr swarm.Address) (*Stamp, error) {
+func (st *stamper) Stamp(addr, idAddr swarm.Address) (*Stamp, error) {
```

Comment:
It would be good to explain the purpose of arguments as there are two addresses now.

Reply:
added comment to the interface, in the close future request address may be not defined.

Reply:
Thanks. It would be very nice to have full sentences with correct capitalization and dots at the end so that it is nicely readable on https://pkg.go.dev/github.com/ethersphere/bee. This is more of a general comment as well.

Reply:
got it, latest changes have this.